userstorage/backend.py

```diff
@@ -93,4 +93,6 @@ def teardown(self):
     # Displaying.
 
     def __str__(self):
-        return self.name
+        if self.name is None:
+            raise TypeError("backend name is NoneType")
+        return str(self.name)
```

Comment:
raising TypeError in __str__ is strange. A backend must have a name, so it
would make sense to raise when creating a backend without a name, not when
calling str(backend) or formatting it log messages. But I don't think we have
a good way to do this and documenting the fact that a backend must define
a name is good enough.

If we want to help pylint we can change the default to empty string.